### ISSUE TITLE
Try to run the simplest rules at https://editor.gorules.io/, but get "Unexpected Application Error!"

### ISSUE THREAD

**Issue by alec-liu** [gorules/editor#24]
Step 1: new graph as JSON below
```
{
  "contentType": "application/vnd.gorules.decision",
  "nodes": [
    {
      "type": "inputNode",
      "id": "e10b25ef-6b4f-415b-a652-14d52b9202c5",
      "name": "request",
      "position": {
        "x": 520,
        "y": 130
      }
    },
    {
      "type": "outputNode",
      "id": "fa9c1488-4ce5-47e2-959b-89e515ba6273",
      "name": "response",
      "position": {
        "x": 960,
        "y": 120
      }
    }
  ],
  "edges": [
    {
      "id": "35e268ac-09d7-4879-863a-61d21515207c",
      "sourceId": "e10b25ef-6b4f-415b-a652-14d52b9202c5",
      "type": "edge",
      "targetId": "fa9c1488-4ce5-47e2-959b-89e515ba6273"
    }
  ]
}%

```

Step 2: try to run the Simulator with {key:"yes"}

Then it shows:
```
Unexpected Application Error!
r.walk(...).forEach is not a function
TypeError: r.walk(...).forEach is not a function
    at https://editor.gorules.io/assets/index-DpMM05Ik.js:6672:26657
    at produce (https://editor.gorules.io/assets/index-DpMM05Ik.js:2003:19920)
    at LIn (https://editor.gorules.io/assets/index-DpMM05Ik.js:6672:26638)
    at https://editor.gorules.io/assets/index-DpMM05Ik.js:6672:26176
    at https://editor.gorules.io/assets/index-DpMM05Ik.js:2003:21931
    at Set.forEach (<anonymous>)
    at Function.i [as setState] (https://editor.gorules.io/assets/index-DpMM05Ik.js:2003:21920)
    at https://editor.gorules.io/assets/index-DpMM05Ik.js:6672:26006
    at https://editor.gorules.io/assets/index-DpMM05Ik.js:2003:21931
    at Set.forEach (<anonymous>)
```
